solidity/random-beacon/contracts/test/SortitionPoolStub.sol

```diff
@@ -2,20 +2,40 @@
 
 pragma solidity ^0.8.6;
 
-import "@keep-network/sortition-pools/contracts/SortitionTree.sol";
+import "./SortitionTreeStub.sol";
 import "../RandomBeacon.sol";
 
 // Stub contract used in tests
-contract SortitionPoolStub is ISortitionPool, SortitionTree {
+contract SortitionPoolStub is ISortitionPool {
```

Comment:
I'd really love to eventually get rid of this stub completely. cc @nkuba @dimpar @tomaszslabon

Reply:
Posted a TODO about that in https://github.com/keep-network/keep-core/pull/2707/commits/c97dba585f021ce3e8ec7fb37e2665a4f7577541 as part of #2707